File: src/main/java/com/procurement/submission/SubmissionApplication.java
package com.procurement.submission;

import com.procurement.submission.config.ApplicationConfig;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication(
    scanBasePackageClasses = ApplicationConfig.class
)
@EnableEurekaClient
public class SubmissionApplication {
    public static void main(final String[] args) {
        SpringApplication.run(SubmissionApplication.class, args);
    }
}

File: src/main/kotlin/com/procurement/submission/SubmissionApplication.kt
package com.procurement.submission

import com.procurement.submission.infrastructure.configuration.ApplicationConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackageClasses = [ApplicationConfig::class])
class SubmissionApplication

fun main(args: Array<String>) {
    runApplication<SubmissionApplication>(*args)
}
